ensime_shared/ensime.py

```diff
@@ -738,7 +747,7 @@ def trigger_callbacks(_json):
                 now = time.time()
             else:
                 result = self.queue.get(False)
-                self.log("unqueue: result received {}".format(str(result)))
+                self.log.debug('unqueue: result received\n%s', result)
```

Comment:
I'm not quite sure what type `result` is, but this looks like a good candidate for pretty printing as well.